README.md

```diff
@@ -39,5 +39,5 @@ cargo test
 ```
 ##TODO
 ### [0.0.1]
-- [ ] [MAID-1231](https://maidsafe.atlassian.net/browse/MAID-1321) safe_ffi - Make reading for file relative to serice home dir
-- [ ] [MAID-1232](https://maidsafe.atlassian.net/browse/MAID-1322) safe_ffi - changes according to other crates and update test cases
+- [X] [MAID-1231](https://maidsafe.atlassian.net/browse/MAID-1321) safe_ffi - Make reading for file relative to serice home dir
```

Comment:
The task Id is wrong in the TODO. Task Id is different in the link and in the text